ding_library.make

```diff
@@ -80,6 +80,7 @@ projects[opening_hours][download][url] = "http://git.drupal.org/project/opening_
 projects[opening_hours][download][revision] = "81146d1e8ab63ca70976596d928e4ec46dfdfd57"
 projects[opening_hours][patch][] = "http://drupal.org/files/issues/is-string-check-2260505-2.patch"
 projects[opening_hours][patch][] = "http://drupal.org/files/issues/change-path-2270935-2.patch"
+projects[opening_hours][patch][] = "http://www.drupal.org/files/issues/Issue620-single-minute-opening-hours.patch"
```

Comment:
In general all patches in make files should have a comment explaining why they are necessary.

If they have been submitted to an issue should they should contain a link to the issue as well. Sometimes this can be inferred from the patch file name. This is not the case here.